packages/amplication-e2e-tester/tests/functions.ts

```diff
@@ -34,7 +34,8 @@ export async function signUp(
   await (
     await page.waitForXPath(`//button[contains(text(),'${continueBtn}')]`)
   ).click();
-  return page.waitForNavigation();
+  await page.waitForNavigation();
+  return page.waitFor(3000);
```

Comment:
Why did you add a wait?

Reply:
sometimes tests using this function randomlly fail because they cant find the xpath of the create new app element, this ensure it doesnt happen